test/logoutresponse_test.rb

```diff
@@ -6,35 +6,38 @@
 class RubySamlTest < Minitest::Test
 
   describe "Logoutresponse" do
+
+    let(:valid_logout_response_without_settings) { OneLogin::RubySaml::Logoutresponse.new(valid_logout_response_document) }
+    let(:valid_logout_response) { OneLogin::RubySaml::Logoutresponse.new(valid_logout_response_document, settings) }
+    #let(:in_relation_to_request_id) { random_id }
+    #let(:valid_logout_response_with_responseid) { OneLogin::RubySaml::Logoutresponse.new(valid_logout_response_document{:uuid => in_relation_to_request_id}, settings) }
```

Comment:
We should remove these 2 commented lines unless we're going to need them soon (if so, we should put a `TODO` comment explaining what's to come)